base/setup/lib/utils/osdetect.c

```diff
@@ -607,16 +607,15 @@ FindExistingNTOSInstall(
 
 static PNTOS_INSTALLATION
 AddNTOSInstallation(
-    IN PGENERIC_LIST List,
-    IN PCWSTR InstallationName,
-    IN USHORT Machine,
-    IN PCWSTR VendorName,
-    IN PCWSTR SystemRootArcPath,
-    IN PUNICODE_STRING SystemRootNtPath, // or PCWSTR ?
-    IN PCWSTR PathComponent,    // Pointer inside SystemRootNtPath buffer
-    IN ULONG DiskNumber,
-    IN ULONG PartitionNumber,
-    IN PPARTENTRY PartEntry OPTIONAL)
+    _In_ PGENERIC_LIST List,
+    _In_ PCWSTR InstallationName,
+    _In_ USHORT Machine,
+    _In_ PCWSTR VendorName,
+    _In_ PCWSTR SystemRootArcPath,
+    _In_ PUNICODE_STRING SystemRootNtPath, // or PCWSTR ?
+    _In_ PCWSTR PathComponent,    // Pointer inside SystemRootNtPath buffer
+    _In_ ULONG DiskNumber,
+    _In_ ULONG PartitionNumber)
```

Comment:
How does the removal of the "PartEntry OPTIONAL" work now? Is it handles elsewhere?

Reply:
Yes, it is handled at lines 196+ above.
(Looking again at that code I could move this into the AddNTOSInstallation() implementation... I'll see later whether this could be useful.)